pkgs/servers/antennas/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, mkYarnPackage, fetchFromGitHub, nodejs }:
+
+mkYarnPackage rec {
+  pname = "antennas";
+
+  version = "3.1+5e1f7375";
```

Comment:
```suggestion
  pname = "antennas";
  version = "3.1+5e1f7375";
```
Please use the unstable format.

Reply:
@SuperSandro2000 What you mean by unstable format?

Reply:
unstable format is `version = "unstable-YYYY-DD-MM"`